model/Licensing/Licensing.md

```diff
@@ -45,11 +45,11 @@ types of Software Artifacts. For example:
     different File (e.g., comment at top of File if it is not within the
     Snippet, LICENSE file in the top directory of a repository)
 
-A declaredLicense is related to a NoneLicenseindicates that the
+A declaredLicense is related to a NoneLicense and indicates that the
```

Comment:
I think this should say "A declaredLicense relationship to a NoneLicense indicates..."

"related" here is (I assume) referring to the use of a Relationship to the NoneLicense value.